mathics/builtin/files.py

```diff
@@ -712,7 +715,17 @@ def apply(self, channel, types, evaluation, options):
 
         # Wrap types in a list (if it isn't already one)
         if not types.has_form("List", None):
+            if types.get_head_name() == "System`Hold":
```

Comment:
This seems hoaky since the speck seems to allow any function and not just `Hold[]` right?